.eslintrc.js

```diff
@@ -123,6 +123,7 @@ module.exports = {
     'no-proto': 'error',
     'no-redeclare': 'error',
     'no-restricted-modules': ['error', 'sys'],
+    /* eslint-disable max-len */
```

Comment:
Is it possible to use `// eslint-disable-next-line max-len` in the right place instead of this blanket disable?